src/atoms/second_order_cone/power_to_socp.jl

```diff
@@ -39,7 +38,7 @@ mutable struct InequalityExpression
   u_var_ind::Int
 end
 
-struct SimpleInequalityExpression
+mutable struct SimpleInequalityExpression
```

Comment:
why does this need to be mutable now?

Reply:
That's a good question. @molet, that was one of your changes--can you explain your reasoning?